### ISSUE TITLE
Search doesn't seem to work in template literals

### ISSUE THREAD

**Issue by LeoLozes** [microsoft/vscode#77925]
When searching for a string that is in a template literal, VS Code Insider doesn't throw any matches:

![image](https://user-images.githubusercontent.com/11133817/61863018-1aa51300-aecf-11e9-9d5b-8061efe4d972.png)

 If I open the file containing the template literal, it works:

![image](https://user-images.githubusercontent.com/11133817/61863067-327c9700-aecf-11e9-9f60-6b6493a7b17a.png)

- VSCode Version: 1.37.0-insider
- OS Version: Windows 10.0.18362

Steps to Reproduce:

1. Search for an expression that is in a template literal and not in an open file. 
2. Search doesn't work.

Does this issue occur when all extensions are disabled?: Yes

**Comment by LeoLozes** [comment]
After reviewing it with the normal VS Code and other editors, it seems that the problem is that the folder path gets too long.

**Comment by roblourens** [comment]
I think it's really unlikely that this has anything to do with template literals, can you show that it finds other matches in this file but not that one?

> it seems that the problem is that the folder path gets too long.

Can you explain some more?

**Comment by LeoLozes** [comment]
> Can you explain some more?

Yes, sorry! It was a project with stupidly long paths because it followed a nested-component folder structure. The file I opened in the second screenshot was actually in a path of 460 characters (!).

I first blamed the insider build (and created this issue a bit hastily, sorry), then I switched to the normal one and the behaviour was the same, so I checked with the "find in files" option of Notepad++ (which is always kind of the last-resort-solution). 
When even Notepad++ couldn't find anything I started checking the length of the path of the file (I remembered [some problems with NTFS and paths larger than 260 chars](https://www.howtogeek.com/266621/how-to-make-windows-10-accept-file-paths-over-260-characters/)). 

I copied the file to a folder closer to the source folder and searched again and sure enough everything worked just fine ...
